bin/i18n-codeorg/in.sh

```diff
@@ -56,8 +56,12 @@ done
 ### Oceans tutorial
 cp_in apps/node_modules/@code-dot-org/ml-activities/i18n/oceans.json i18n/locales/source/blockly-mooc/fish.json
 
-### AI Lab datasets
+### AI Lab UI Strings
+source_ui=apps/node_modules/@code-dot-org/ml-playground/i18n/ailab.json
+staged_ui=i18n/locales/source/blockly-mooc/ailab.json
```

Comment:
The [apps/i18n/ailab/en_us.json](https://github.com/code-dot-org/code-dot-org/blob/425b122757673e82564bc1968300b0c01490b902/apps/i18n/ailab/en_us.json) gets sync in with the following command as well:
`### Apps
orig_dir=apps/i18n
loc_dir=i18n/locales/source/blockly-mooc
mkdir -p $loc_dir`
Will this override changes if strings are added to the the json file in the apps directory?